cmd/thanos/query.go

```diff
@@ -125,6 +125,9 @@ func registerQuery(app *extkingpin.App) {
 
 	defaultEvaluationInterval := extkingpin.ModelDuration(cmd.Flag("query.default-evaluation-interval", "Set default evaluation interval for sub queries.").Default("1m"))
 
+	defaultRangeQueryStep := extkingpin.ModelDuration(cmd.Flag("query.default-step", "Set default step for range queries").
```

Comment:
I would add more context. For example the fact, that this works well if step is not set and we made Thanos UI doing that. This will not work from Grafana, but Grafana has this `__step` variable which can be used (although not sure how close to this it works).

Reply:
Understood. I will write proper description of the flag.